applications/CoSimulationApplication/tests/mpm_dem/MaterialPointMaterials.json

```diff
@@ -7,11 +7,11 @@
                 "name" : "LinearElasticIsotropicPlaneStrain2DLaw"
             },
             "Variables"        : {
-                "THICKNESS"             : 1.0,
-                "PARTICLES_PER_ELEMENT" : 3,
-                "DENSITY"               : 1000.0,
-                "YOUNG_MODULUS"         : 90000000.0,
-                "POISSON_RATIO"         : 0.0
+                "THICKNESS"                     : 1.0,
+                "MATERIAL_POINTS_PER_ELEMENT"   : 3,
```

Comment:
this is another silent behavior change, I dont see any migration from old to new input. You could do this in the analysis stage of the MPM-app. I.e. read the materials file and if necessary update the variable names

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/855f9891ec95d4953cd32536ab939a073efa2acb/applications/MPMApplication/python_scripts/mpm_solver.py#L493-L505

As you suggested, I added the compatibility with the old input `PARTICLES_PER_ELEMENT` here.